docs/developer-tool-faq.md

```diff
@@ -523,15 +523,15 @@ git config --global core.safecrlf false
 
 ### How do I upgrade docker image in a pre-commit hook?
 
-1. Identify the docker image tag for `ibmcom/detect-secrets-hook` in your `.pre-commit-config.yaml` file. For example, the default is `latest`.
+1. Identify the docker image tag for `icr.io/git-defenders/detect-secrets-hook` in your `.pre-commit-config.yaml` file. For example, the default is `latest`.
 1. If you are using a latest tag such as `latest`.
-    1. In a terminal, run `docker pull ibmcom/detect-secrets-hook:latest` to get the latest image.
+    1. In a terminal, run `docker pull icr.io/git-defenders/detect-secrets-hook:latest` to get the latest image.
 1. If you are using a specific version tag, such as `0.13.1+ibm.37.dss`
     1. Update the `.pre-commit-config.yaml` to use a newer version tag.
 
 ### Can I pull a specific version of docker image?
 
-Yes, any tag listed [in docker hub for image ibmcom/detect-secrets](https://hub.docker.com/repository/docker/ibmcom/detect-secrets) can be used. You can use the same approach to find tags for `ibmcom/detect-secrets-hook`.
+Yes, any tag listed [in docker hub for image icr.io/git-defenders/detect-secrets](https://hub.docker.com/repository/docker/icr.io/git-defenders/detect-secrets) can be used. You can use the same approach to find tags for `icr.io/git-defenders/detect-secrets-hook`.
```

Comment:
I don't think thats the correct link

Reply:
Ah never mind, you're talking about https://hub.docker.com/repository/docker/icr.io/git-defenders/detect-secrets 🤦, my bad. Users don't have access to a GUI for viewing the IBM Cloud Container Registry images. They would have to use the IBM Cloud CLI Container Registry plugin to list tags.

Reply:
ahh when I did the changes the first time I had the change right but I forgot the second time through- will fix